lib/skate_web/channels/notifications_channel.ex

```diff
@@ -36,6 +36,6 @@ defmodule SkateWeb.NotificationsChannel do
 
     initial_notifications = notification_fetch.(username)
 
-    {:ok, %{initial_notifications: initial_notifications}, socket}
+    {:ok, %{data: %{initial_notifications: initial_notifications}}, socket}
```

Comment:
Great call. I'll add a temporary change to the `useChannel` hook to handle the old way of not getting `data:`

Reply:
That should handle the case where a client with the new version of the front-end code is talking to a server with the old version of the back-end code. If a client with the old version of the front-end code is talking to a server with the new version of the back-end code I think that still might be a potential issue?

Reply:
Absolutely true! New idea: 
1. first deploy a change to backend that has the payload `%{data: %{initial_notifications: initial_notifications}, initial_notifications: initial_notifications}`
2. deploy this PR & change the backend payload to `%{data: %{initial_notifications: initial_notifications}}`
Does that seem reasonable?

Reply:
Sounds good to me.

Reply:
Hmm actually regarding step 2, I think we only want to remove the old `initial_notifications` key _after_ the front-end changes in this PR go out to all users, so it would have to be another PR / release after this one. I'm not sure if that changes anything else about the process, but I'll think about it. It may be possible to do in two deploys rather than three.